source/dayTime.cpp

```diff
@@ -0,0 +1,66 @@
+#include "../headers/dayTime.h"
+#include <iostream>
+
+int DayTime::stringToDays(string input) {
+	int day = -1;
+
+	input = stringToLower(input);
+
+	if (stringToLower(input) == "mw") day = 0;
+	if (stringToLower(input) == "tr") day = 1;
+
+	return day;
+}
+
+int DayTime::stringToHour(string input) {
+	int hour = -1;
+
+	input = stringToLower(input);
+
+	if (input == "8 am" || input == "8:00 am" || input == "08:00 am" || input == "8" || input == "8:00" || input == "08:00") hour = 0;
+	if (input == "9:40 am" || input == "09:40 am" || input == "9:40" || input == "09:40") hour = 1;
+	if (input == "11:20 am" || input == "11:20") hour = 2;
+	if (input == "1 pm" || input == "1:00 pm" || input == "01:00 pm" || input == "13" || input == "13:00") hour = 3;
+	if (input == "2:40 pm" || input == "02:40 pm" || input == "14:40") hour = 4;
+	if (input == "4:20 pm" || input == "04:20 pm" || input == "16:20") hour = 5;
+
+	return hour;
+}
+
+string DayTime::stringToLower(string input) {
```

Comment:
C++ has a lowercase conversion function

Reply:
The only functions I found where toupper() and tolower(), that only accept char inputs. Do you have a function that works with strings?

Reply:
See the top answer here for two different options on strings: https://stackoverflow.com/questions/3403844/tolower-function-for-c-strings

Reply:
We would need to install boost-algorithm. Should we?

Reply:
No. The only place you're using this is in the stringToX() functions, and we don't need those either given our numerical formatting. stringToHour, stringToDays and stringToLower can all go away

Reply:
[Use this.](https://www.geeksforgeeks.org/conversion-whole-string-uppercase-lowercase-using-stl-c/ )

Reply:
^ These functions are only used internally as indices in the dayTime array. Because of that we can simply say "use 0 when you are setting the 8:00 am time slot" etc. instead of having to use these conversions each time